.rubocop_todo.yml

```diff
@@ -266,7 +266,7 @@ Metrics/AbcSize:
 # Configuration parameters: CountComments, CountAsOne, ExcludedMethods, AllowedMethods, AllowedPatterns, IgnoredMethods, inherit_mode.
 # AllowedMethods: refine
 Metrics/BlockLength:
-  Max: 210
+  Max: 212
```

Comment:
Just out of curiosity: What are these extra two characters required for?

Reply:
I don't rember 100% but I think it's the swagger_docs block in the swagger_helper.rb ... and it's huge.
Maybe there is a way to split it up, but before rswag everything of that was a huge block in swagger.yml